controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2300 @@
+    title: Ensure iptables firewall rules exist for all open ports (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7190
+  - id: 3.4.4.1.5
+    title: Ensure iptables is enabled and active (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5257
+  - id: 3.4.4.2.1
+    title: Ensure ip6tables default deny firewall policy (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5258
+  - id: 3.4.4.2.2
+    title: Ensure ip6tables loopback traffic is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 3.4.4.2.3
+    title: Ensure ip6tables outbound and established connections are configured (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7191
+  - id: 3.4.4.2.4
+    title: Ensure ip6tables firewall rules exist for all open ports (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7192
+  - id: 3.4.4.2.5
+    title: Ensure ip6tables is enabled and active (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 3.5
+    title: Ensure wireless interfaces are disabled (Automated)
+    levels:
+      - l1_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - wireless_disable_interfaces
+
+  - id: 3.6
+    title: Disable IPv6 (Manual)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - kernel_module_ipv6_option_disabled
+
+  - id: 4.1.1.1
+    title: Ensure auditd is installed (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - package_audit_installed
+
+  - id: 4.1.1.2
+    title: Ensure auditd service is enabled (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - service_auditd_enabled
+
+  - id: 4.1.1.3
+    title: Ensure auditing for processes that start prior to auditd is enabled (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - grub2_audit_argument
+
+  - id: 4.1.1.4
+    title: Ensure audit_backlog_limit is sufficient (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - grub2_audit_backlog_limit_argument
+
+  - id: 4.1.2.1
+    title: Ensure audit log storage size is configured (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - auditd_data_retention_max_log_file
```

Comment:
I agree. I've set this to 6, since that seems to be the value used by a lot of other profiles, and it seems to be the default. I personally keep a lot more on my systems but not changing this value will help avoid breaking anyone's deployment pipelines if they're not creating very large `/var/log/audit` volumes.